directories/src/utils/encryption.js

```diff
@@ -49,8 +49,18 @@ const decrypt = (text, secretKey) => {
   }
 };
 
+const decryptTest = (text, secretKey) => {
+  //TODO temporary for debugging
+  try {
+    return CryptoJS.AES.decrypt(text, secretKey).toString();
+  } catch (e) {
+    return "";
+  }
+};
```

Comment:
Why not just use the normal decrypt function?

Reply:
The normal decrypt function uses the UTF8 representation of the treasure - for debugging reasons I just output the hex string since the decryption is not working right now

Reply:
Ah, I see.  I didn't look at the methods closely enough.

Reply:
And I didn't want to change the original function since it is being used in different places as well